resources/js/components/VerificationForm.vue

```diff
@@ -59,19 +63,32 @@
                     code: '',
                     token: ''
                 }),
-                errorMessage: ''
+                errorMessage: '',
+                resend_timeout: 30,
+                intervalHandler: null
             }
         },
         props: [ 'token', 'mobile' ],
         created() {
             this.form.token = this.token;
+            this.intervalHandler = setInterval(() => {
+                if (this.resend_timeout)
+                    this.resend_timeout--
+            }, 1000)
+        },
+        watch: {
+            'form.code'(value) {
+                if (!/^\d{1,6}$/.test(value))
+                    this.form.code = value.slice(0, -1)
+            }
         },
         methods: {
             async verify() {
                 this.errorMessage = '';
                 await this.$axios.get('/sanctum/csrf-cookie')
 
                 this.form.post('/api/verify').then((response) => {
+                    clearInterval(this.intervalHandler);
```

Comment:
clearInterval should be called as soon as the timer reached 😄. So you need to call this inside Line Line 74